src/cms/urls.py

```diff
@@ -291,6 +291,18 @@
         r"^ajax/",
         include(
             [
+                url(
+                    r"^render/",
+                    include(
+                        [
+                            url(
+                                r"^mirrored_page_field/",
+                                pages.render_mirrored_page_field,
+                                name="render_mirrored_page_field",
+                            ),
+                        ]
+                    ),
+                ),
```

Comment:
Why is it necessary to use ajax here? The only case I can imagine is, when the user opens the form and wants keep track of changes to all other regions pages without refreshing, is this right?